ambry-cloud/src/test/java/com/github/ambry/cloud/azure/AzureBlobDataAccessorTest.java

```diff
@@ -96,10 +81,26 @@ static BlockBlobClient setupMockBlobClient(BlobServiceClient mockServiceClient)
     lenient().when(mockBlobProperties.getMetadata()).thenReturn(metadataMap);
     Response<BlobProperties> mockPropertiesResponse = mock(Response.class);
     lenient().when(mockPropertiesResponse.getValue()).thenReturn(mockBlobProperties);
-    lenient().when(mockBlockBlobClient.getPropertiesWithResponse(any(), any(), any())).thenReturn(mockPropertiesResponse);
+    lenient().when(mockBlockBlobClient.getPropertiesWithResponse(any(), any(), any()))
+        .thenReturn(mockPropertiesResponse);
     return mockBlockBlobClient;
   }
 
+  @Before
```

Comment:
Was there a reason to move this method?  My understanding is public methods go before package private ones.

Reply:
fixed it. This was the job of Intellij's code reformatter.